monitoring-engine/src/main/java/io/opentelemetry/exporter/otlp/internal/LogAdapter.java

```diff
@@ -0,0 +1,114 @@
+package io.opentelemetry.exporter.otlp.internal;
```

Comment:
I understand it's bad manners. I'm sure I'll remove this class when this [PR](https://github.com/open-telemetry/opentelemetry-java/pull/3415) is merged.